frontend/apps/filemanager/filemanagershortcuts.lua

```diff
@@ -50,15 +45,29 @@ function FileManagerShortcuts:updateItemTable()
         })
     end
 
+    table.sort(item_table, function(l, r)
+        return l.text < r.text
+    end)
+    local item_table_sorted = {}
+    table.insert(item_table_sorted, {
+        text = _("Add new folder shortcut"),
+        callback = function()
+            self:addNewFolder()
+        end,
+    })
+    for _, item in ipairs(item_table) do
+        table.insert(item_table_sorted, item)
+    end
```

Comment:
I have no idea what's going on here tbh. ;-) Why not just:

```diff
diff --git a/frontend/apps/filemanager/filemanagershortcuts.lua b/frontend/apps/filemanager/filemanagershortcuts.lua
index 3f54acce..8fa5d113 100644
--- a/frontend/apps/filemanager/filemanagershortcuts.lua
+++ b/frontend/apps/filemanager/filemanagershortcuts.lua
@@ -48,16 +48,12 @@ function FileManagerShortcuts:updateItemTable()
     table.sort(item_table, function(l, r)
         return l.text < r.text
     end)
-    local item_table_sorted = {}
-    table.insert(item_table_sorted, {
+    table.insert(item_table, 1, {
         text = _("Add new folder shortcut"),
         callback = function()
             self:addNewFolder()
         end,
     })
-    for _, item in ipairs(item_table) do
-        table.insert(item_table_sorted, item)
-    end
 
     -- try to stay on current page
     local select_number
@@ -67,7 +63,7 @@ function FileManagerShortcuts:updateItemTable()
     end
 
     self.fm_bookmark:switchItemTable(nil,
-        item_table_sorted, select_number)
+        item_table, select_number)
 end
 
 function FileManagerShortcuts:addNewFolder()
```

Reply:
I simply wasn't to aware of the possibility to insert at the start of a table. I've pushed a commit implementing your suggestion.